app/uk/gov/hmrc/customsservicestatusfrontend/views/DashboardPage.scala.html

```diff
@@ -65,6 +68,9 @@ <h1 class="govuk-heading-l">@messages("dashboard.h1")</h1>
 @unavailableContent = {
     <h1 class="govuk-heading-l">@messages("dashboard.h1")</h1>
     <p class="govuk-body">@Html(messages("dashboard.p1", s"<b>${Formatters.instantFormatHours(Instant.now)}</b>"))</p>
+    <label class="govuk-label govuk-visually-hidden" for="gvms-status">
+        @messages("dashboard.status.screen.reader")
+    </label>
```

Comment:
This is a generic service, not specific to GVMS, so:

1. the message key needs to include the id of the service concerned (and that part of the key itself will be dynamically resolved)
2. we should change the id of whichever element this label is for so it does not include "gvms" (this appears to be a pre-existing element id, but would be grateful if you could correct).

Reply:
updated.